website/snippets/core-versions-table.md

```diff
@@ -1,10 +1,12 @@
-| dbt Core                        | Initial Release | Active Support Until | Critical Support Until  | dbt Cloud Until | Final Patch |
-|---------------------------------|-----------------|----------------------|-------------------------|-----------------|-------------|
-| **v0.X**                        | (various dates) | v1.0.0 release       | Dec 3, 2021             | See below ⚠️     | v0.21.1     |
-| [**v1.0**](upgrading-to-v1.0)   | Dec 3, 2021     | v1.1.0 release       | Dec 3, 2022             | Dec 2022        |             |
-| [**v1.1**](upgrading-to-v1.1)   | Apr 28, 2022    | v1.2.0 release       | Apr 28, 2023            | Apr 2023        |             |
-| [**v1.2**](upgrading-to-v1.2)   | Jul 26, 2022    | v1.3.0 release       | Jul 26, 2023            | Jul 2023        |             |
-| [**v1.3**](upgrading-to-v1.3)   | Oct 12, 2022    | v1.4.0 release       | Oct 12, 2023            | Oct 2023        |             |
-| _**v1.4**_                      | _Jan 2023_      | _v1.5.0 release_     | _Jan 2024_              | _Jan 2024_      |             |
-
-_Italics: Future releases, NOT definite commitments. Shown for indication only._
+| dbt Core                        | Initial Release | Active Support Until | Critical Support Until  | dbt Cloud Until |
+|---------------------------------|-----------------|----------------------|-------------------------|-----------------|
+|  **v0.X**                       | (Various dates) | v1.0 release         | Deprecated              | Deprecated      |
+| [**v1.0**](upgrading-to-v1.0)   | Dec 3, 2021     | v1.1.0 release       | Dec 3, 2022             | Dec 2022        |
+| [**v1.1**](upgrading-to-v1.1)   | Apr 28, 2022    | v1.2.0 release       | Apr 28, 2023            | Apr 2023        |
+| [**v1.2**](upgrading-to-v1.2)   | Jul 26, 2022    | v1.3.0 release       | Jul 26, 2023            | Jul 2023        |
+| [**v1.3**](upgrading-to-v1.3)   | Oct 12, 2022    | v1.4.0 release       | Oct 12, 2023            | Oct 2023        |
+| _**v1.4**_                      | Jan 25, 2023    | v1.5.0 release       | Jan 25, 2024            | Jan 2024        |
```

Comment:
This will want a link to the v1.4 migration guide once it exists (https://github.com/dbt-labs/docs.getdbt.com/pull/2646)